### DISCUSSION TITLE
Since in config file

### DISCUSSION THREAD

**Discussion by andersonvcv** [stryker-mutator/stryker-net#2639] [Q&A] [answered]
Hello,

I'm trying to use the configuration file to facilitate the local run of Stryker, but I'm having an issue with the "since" option. 
Is there a way to automatically capture the current branch (without having to manually insert the hash) and run only on the changes? For me, the goal of the configuration file is just to run "dotnet stryker" and see the results without configuring anything...

It would be something like:
  "stryker-config": {
    "compare-against-brach": "main"
  }
It can then make use of "git branch --show-current" to get the current branch and "git merge-base compare-against-brach current-branch " to compute the hash.

**Comment by rouke-broersma** [comment]
We already have a default compare branch, which is master. There is no good concept of "default" branch in git so we use the up until recently defacto default of "master". Since the default has switched to main, but a significant, probably the largest part of the ecosystem still user master instead of main we have not yet changed this. Using two defaults is currently not something we can support cleanly and afaik there is no canonical answer for getting the default branch, certainly not using libgit2sharp which we use. 

We also already use the current tip of the current branch as the compare so you don't need to set it.

**Comment by andersonvcv** [comment]
_"We also already use the current tip of the current branch as the compare so you don't need to set it."_
Can we already run only on the changes without specifying the current-branch or master? How can I do it?

**Reply by rouke-broersma** [reply, marked as answer]
Just don't specify them?

**Reply by andersonvcv** [reply]
Oh, awesome! Thanks for the quick replies!
